lib/builder-browserify.js

```diff
@@ -25,12 +28,49 @@ function configure (bundler, cfg) {
     }
 }
 
-module.exports = function (files, config, cb) {
+// the builder API is var build = require('builder')(); build(files, config, cb)
+// So that we can have multiple different `watchifyBundler` caches per build instance.
+module.exports = function() {
+    var watchifyBundler;
+
+    return function(files, config, cb) {
```

Comment:
We could also deal with this using a cache map based on files + config hash but that would require a lot of work without any seen benefit but not breaking the builder API (and custom builders if used?)